### PR TITLE
Update supported platforms for Kubernetes App and EKS control plane App

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update supported platforms for Kubernetes App and EKS control plane App

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

File: docs/integrations/containers-orchestration/kubernetes.md [hunk 1]
```diff
@@ -21,16 +21,8 @@ For an end-to-end solution for deploying, managing, monitoring, and administerin
...
-* Kubernetes
-   * 1.10 and later
-* Kops
-   * 1.13.10-k8s
-   * 1.13.10-kops
-   * 1.12.8-k8s
-   * 1.12.2-kops
-   * 1.10.13-k8s
-   * 1.10.0-kops
+The Sumo Logic Kubernetes App is compatible with [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
```
Comment (stacykor) [inline docs/integrations/containers-orchestration/kubernetes.md:24]:
```suggestion
The Sumo Logic Kubernetes App is compatible with the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
```

File: docs/integrations/containers-orchestration/kubernetes.md [hunk 2]
```diff
@@ -21,16 +21,8 @@ For an end-to-end solution for deploying, managing, monitoring, and administerin
...
+List of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```
Comment (stacykor) [inline docs/integrations/containers-orchestration/kubernetes.md:25]:
```suggestion
A list of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```

**Review by stacykor — CHANGES_REQUESTED** [review]
Not sure about pulling all of this into other reference docs. It sends the user back and forth. Any way we could just call this information from the other repository?

**Comment by kkujawa-sumo** [comment]
> Not sure about pulling all of this into other reference docs. It sends the user back and forth. Any way we could just call this information from the other repository?

Unfortunately the only place where we update supported versions on time is https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix 🤷 

Now we have very old version mention in our documentation, Kuberentes 1.10 is no longer supported and for sure this version contains lots of changes comparing with newer versions of Kuberentes.

**Comment by JainM6** [comment]
@stacykor: Could you please review @kkujawa-sumo's response?  If there are no further changes, we can proceed with the update and close the request.

File: docs/integrations/amazon-aws/eks-control-plane.md [hunk 3]
```diff
@@ -13,28 +13,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-   </td>
-  </tr>
-  <tr>
-   <td>Amazon EKS
-   </td>
-   <td>1.13.8, 1.11.10
-   </td>
-  </tr>
-</table>
+List of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/eks-control-plane.md:16]:
```suggestion
The list of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```

File: docs/integrations/containers-orchestration/kubernetes.md [hunk 4]
```diff
@@ -21,16 +21,8 @@ For an end-to-end solution for deploying, managing, monitoring, and administerin
...
-   * 1.10 and later
-* Kops
-   * 1.13.10-k8s
-   * 1.13.10-kops
-   * 1.12.8-k8s
-   * 1.12.2-kops
-   * 1.10.13-k8s
-   * 1.10.0-kops
+The Sumo Logic Kubernetes App is compatible with the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
+A list of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/kubernetes.md:25]:
```suggestion
The Sumo Logic Kubernetes App is compatible with the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection). A list of supported platforms can be found [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs#support-matrix).
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**